harvester.py

```diff
@@ -20,10 +20,12 @@ def read_xml(terminology):
     OUT: ET root object
     '''
     url = terminology['uri']
+    uri_postfix = read_config_uriPostfix(config_file_name)
+    url = url + uri_postfix
     collection_name = terminology['collection_name']
     try:
         head = requests.head(url)
-        if head.headers['Content-Type'] == 'application/rdf+xml':
+        if head.headers['Content-Type'] == 'application/rdf+xml; charset=utf-8':
```

Comment:
this should maybe more flexible, especially with whitespace. see the next one below!
I'd suggest to just do "starts with application/xml"

Reply:
will do

Reply:
done.